### ISSUE TITLE
Camera Pose？

### ISSUE THREAD

**Issue by WANGLXUE** [openMVG/openMVG#1502]
(1) While in SFM pipline, which file contains the camera pose information?
    I have searched the issues listed. #693 tells somethings but I didn't find any bin file. I think in the SFM pipline, there are only keypoints, descriptors, matches, final .ply 3d model. Alson alongside there is a sfm_data.json file. But I checked the smf_data.json file, there is no information about the R and T information?  Can you help me figure out a way to find the R and T information.

**Comment by pmoulon** [comment]
See https://github.com/openMVG/openMVG/issues/1316#issuecomment-392141887
Once the sfm_data.bin is converted to .json or .xml you will have all the values you need ;-)

**Comment by WANGLXUE** [comment]
I am sorry.
I just follow the steps from "https://openmvg.readthedocs.io/en/latest/software/SfM/SfM/".
And there is no .bin file in the final output. Only two .ply which are "cameraPath_translation_averaging.ply" and "initial_structure.ply".

**Comment by WANGYuSCU** [comment]
I am sorry. I checked the source code of mian_GlobalSfM. I think there are more output such as sfm_data.bin. But I failed to get these information. I will check again. 
I just follow your steps and see the output of each step.
I think GlobalSfM rely on matches.e.bin and IncrementalSfM rely on matches.f.bin. Is that right? So after ComputeMatches, I need to change the matches.f.bin to matches.e.bin manually if I want to use GlobalSfM?
